src/utils/index.ts

```diff
@@ -1,13 +1,36 @@
 import axios, { AxiosError } from "axios";
-import { GELATO_ADDRESSES } from "../constants";
+import { GELATO_ADDRESSES, W3F_API_ENDPOINT } from "../constants";
+import { W3fNetwork } from "../types/W3FNetworks.interface";
 
-export function isAutomateSupported(chainId: number): boolean {
-  return Boolean(GELATO_ADDRESSES[chainId]);
+export async function getNetwork(chainId: number): Promise<W3fNetwork | null> {
+  try {
+    const response = await axios.get<{ network: W3fNetwork }>(
+      `${W3F_API_ENDPOINT}/networks/${chainId}`,
+    );
+
+    return response.data.network;
+  } catch (error) {
+    if (axios.isAxiosError(error)) {
+      if (error.response?.status !== 404) {
+        console.error("Error getting network:", error.message);
+      }
+    } else {
+      console.error("Unexpected error getting network:", error);
+    }
+
+    return null;
+  }
+}
+
+export async function isAutomateSupported(chainId: number): Promise<boolean> {
+  return (await getNetwork(chainId)) !== null;
 }
 
-export function isAutomateDevSupported(chainId: number): boolean {
+export async function isAutomateDevSupported(
+  chainId: number,
+): Promise<boolean> {
   return (
-    isAutomateSupported(chainId) &&
+    (await isAutomateSupported(chainId)) &&
     Boolean(GELATO_ADDRESSES[chainId].automateDev)
```

Comment:
we won't rely on static config file here anymore.
instead we need to see if network is supported on dev api endpoint